File: backend-api/src/main/java/com/smartenergy/backendapi/repository/EstadoEquipoRepository.java
package com.smartenergy.backendapi.repository;

import com.smartenergy.backendapi.model.EstadoEquipo;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface EstadoEquipoRepository extends  JpaRepository<EstadoEquipo, UUID> {
}

File: mobile-app/app/src/main/java/com/example/smartenergy/repository/EstadoEquipoRepository.kt
package com.example.smartenergy.repository

import com.example.smartenergy.model.EstadoEquipo

class EstadoEquipoRepository {

    private val estados = mutableListOf<EstadoEquipo>()

    fun obtenerEstados(): List<EstadoEquipo> = estados

    fun agregarEstado(estado: EstadoEquipo) {
        estados.add(estado)
    }
}

